### ISSUE TITLE
Not all input properties are available as output properties

### ISSUE THREAD

**Issue by MerelyAPseudonym** [pulumi/pulumi-azure-nextgen#111]
I haven’t investigated to see how widespread it is across the codebase, but the example occurrence I came across is in `containerservice`’s `ManagedCluster`. The [documentation](https://www.pulumi.com/docs/reference/pkg/azure-nextgen/containerservice/managedcluster/#outputs) states that “all input properties are implicitly available as output properties”, but `resourceGroupName`, an input property, is not present as an output property. This can also be verified by inspecting [the file](https://github.com/pulumi/pulumi-azure-nextgen/blob/v0.2.4/sdk/nodejs/containerservice/v20200901/managedCluster.ts).

So, is that omission a bug? Or is that claim in the documentation obsolete?

**Comment by mikhailshilkov** [comment]
Good point. Currently, the inputs that are part of the resource path aren't exposed as outputs (because they aren't in the HTTP response from Azure). We'll take a look at reconciling the behavior vs. the docs.

**Comment by XBeg9** [comment]
Same question here, old Azure provider does provide inputs as outputs, which helps a lot to decouple infra logic. Currently, you need to repeat `resourceGroupName` N-times for each subsequent resource creation.
